platform-emf/src/main/java/com/softicar/platform/emf/form/EmfForm.java

```diff
@@ -122,9 +124,25 @@ public void focusFrame() {
 
 	// ------------------------------ behavioral configuration ------------------------------ //
 
+	/**
+	 * Sets a callback {@link Consumer} to be called whenever a new
+	 * {@link IEmfTableRow} was inserted through this {@link EmfForm}.
+	 * <p>
+	 * This callback mechanism is primarily intended for UI updates. It is
+	 * called when the new {@link IDbTableRow} was <b>persistently</b> inserted
+	 * and all transactions were successfully <b>committed</b>.
```

Comment:
I think we should be more explicit here, by adding sth. like:
`The callback should <b>not</b> write to the database.`
Then, in the paragraph below, we (already) explain what to do _instead_.

Reply:
done